### ISSUE TITLE
Error with pickles in Python 3

### ISSUE THREAD

**Issue by ahye** [jeriek/xsec#2]
There seems to be a problem with using the pickled GPs in Python 3. I get the following error trying to run
```
Traceback (most recent call last):
  File "/Users/ahye/Projects/gambit/xsecs/smoking/pyenv3/bin/xsec-test", line 44, in <module>
    xsec.load_processes(processes)
  File "/Users/ahye/Projects/gambit/xsecs/smoking/pyenv3/lib/python3.7/site-packages/xsec/gploader.py", line 391, in load_processes
    process_xstype, energy
  File "/Users/ahye/Projects/gambit/xsecs/smoking/pyenv3/lib/python3.7/site-packages/xsec/gploader.py", line 324, in load_single_process
    gp_reco = reconstruct_gp(model_file)
  File "/Users/ahye/Projects/gambit/xsecs/smoking/pyenv3/lib/python3.7/site-packages/xsec/gploader.py", line 216, in reconstruct_gp
    gp_model = joblib.load(file_object)
  File "/Users/ahye/Projects/gambit/xsecs/smoking/pyenv3/lib/python3.7/site-packages/joblib/numpy_pickle.py", line 575, in load
    obj = _unpickle(fobj)
  File "/Users/ahye/Projects/gambit/xsecs/smoking/pyenv3/lib/python3.7/site-packages/joblib/numpy_pickle.py", line 518, in _unpickle
    raise new_exc
ValueError: You may be trying to read with python 3 a joblib pickle generated with python 2. This feature is not supported by joblib.
```

**Comment by agbuckley** [comment]
Interesting, this is usually a problem the other way around: pickles written using Py3 will be default be in a protocol unknown to Py2. Protocol detection on read should be safe and automatic in Py3. Maybe joblib does things differently: see the bottom of https://joblib.readthedocs.io/en/latest/persistence.html

Or is it possible that it was actually generated using a newer version of Py3 (with later default protocol) than Are is using?

**Comment by jeriek** [comment]
The pickles were created with Py2.7, but since the pickled objects were simple arrays and dictionaries everything seemed to work on Py3 as well. Though apparently not in all environments ... Since the joblib pickle compatibility across Python versions indeed turns out to be unreliable, we'll have a bugfix update up soon with separate data files for Py2 and Py3.

**Comment by jeriek** [comment]
I've just released v1.0.1, which fixes the issue. I converted the Py2 joblib pickles and now there are separate sets of data binaries for Py2 and Py3. The [xsec-download-gprocs](https://github.com/jeriek/xsec/blob/master/scripts/xsec-download-gprocs) script automatically detects which Python version is being used, and downloads the right set of files.
